lib/ansible/module_utils/network/enos/enos.py

```diff
@@ -1,32 +1,21 @@
-# This code is part of Ansible, but is an independent component.
-# This particular file snippet, and this file snippet only, is BSD licensed.
-# Modules you write using this snippet, which is embedded dynamically by
-# Ansible still belong to the author of the module, and may assign their own
-# license to the complete work.
 #
-# Copyright (C) 2017 Lenovo, Inc.
+# Copyright (C) 2018 Lenovo, Inc.
```

Comment:
Same comment as the other PR.  Advancing this copyright date, instead of leaving as is, or using a range, is likely not what you want.